bundles/AdminBundle/public/js/pimcore/settings/docTypes.js

```diff
@@ -149,13 +149,14 @@ pimcore.settings.document.doctypes = Class.create({
             },
             {
                 text: t("priority"),
-                flex: 50,
+                flex: 30,
                 sortable: true,
                 dataIndex: 'priority',
-                editor: new Ext.form.ComboBox({
-                    store: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                editor: new Ext.form.NumberField({
                     mode: "local",
-                    editable: false,
+                    editable: true,
+                    minValue: 1,
```

Comment:
Sorry about this one, but I mentioned it in the review before :) 
If the min value is 1 and the initial value is 0, you cannot change back to the initial value once you set the priority. So I would keep it at 0 tbh.
```suggestion
                    minValue: 0,
```